### DISCUSSION TITLE
Sign added after each task

### DISCUSSION THREAD

**Discussion by arnauldb** [todotxt/todo.txt-cli#321] [Q&A] [answered]
Hello,

On my conky (Linux) I have a sign added  after each task:
![y7i4rzT](https://user-images.githubusercontent.com/13196747/97902286-47341580-1d35-11eb-8dc7-db8a1a94750a.png)

I don't understand why and where it comes from...

Thank you for any help.

**Comment by inkarkat** [comment, marked as answer]
It could be CR-LF (i.e. Windows-style) line endings; a text editor (e.g. in Vim `:setlocal fileformat?`) should be able to tell you, or use `$ file ~/todo/todo.xt` and watch for _with CRLF line terminators_.

The `todo.sh` utility works with LF (Unix) line endings; other clients may create tasks with other line endings, but this often can be configured (e.g. in the _Simpletask_ Android app).

**Comment by arnauldb** [comment]
> It could be CR-LF (i.e. Windows-style) line endings;

Thanks a lot! You were right. My todo.txt is synchronized with my phone (Simpletask app) and in the Simpletask app in settings, there is an option to have Windows-style line endings. Disabling this setting solved the problem.

**Comment by inkarkat** [comment]
Ah, great! I'm closing this issue then.
